test/runtests.jl

```diff
@@ -4,50 +4,44 @@ using SafeTestsets
-    @time @safetestset "Model Construction" begin include("make_model.jl") end
-    @time @safetestset "Custom Functions" begin include("custom_functions.jl") end
-    @time @safetestset "Model Modification" begin include("model_modification.jl") end
-    @time @safetestset "Symbolic Stoichiometry" begin include("symbolic_stoich.jl") end
-    @time @safetestset "Conservation Laws" begin include("conslaws.jl") end
-    @time @safetestset "Network Properties" begin include("network_properties.jl") end
-
-    # Test api
-    @time @safetestset "API" begin include("api.jl") end
-
-    # Tests various core properties of the package.
-    @time @safetestset "Higher Order" begin include("higher_order_reactions.jl") end
-    @time @safetestset "U0 and Parameters Input Types" begin include("u0_n_parameter_inputs.jl") end
-
-    # Tests related to solving Ordinary Differential Equations.
-    @time @safetestset "ODE System Solving" begin include("solve_ODEs.jl") end
-    @time @safetestset "Make Jacobian" begin include("make_jacobian.jl") end
-    #@time @safetestset "DiffEq Steady State Solving" begin include("steady_state_problems.jl") end
-
-    # Tests related to solving Stochastic Differential Equations.
-    @time @safetestset "SDE System Solving" begin include("solve_SDEs.jl") end
-
-    # Tests related to solving Jump Systems.
-    @time @safetestset "Jump System Solving" begin include("solve_jumps.jl") end
-
-    # Tests related to PDESystems
-    @time @safetestset "PDE Systems" begin include("pdes.jl") end
-
-    # Miscellaneous tests
-    #@time @safetestset "Basic Plotting" begin include("plotting.jl") end
-    @time @safetestset "Latexify" begin include("latexify.jl") end
-
+    ### Tests the properties of ReactionSystems. ###
+    @time @safetestset "ReactionSystem Structure" begin include("reactionsystem_structure/reactionsystem.jl") end
+    @time @safetestset "Higher Order Reactions" begin include("reactionsystem_structure/higher_order_reactions.jl") end
+
+    ### Tests model creation via the @reaction_network DSL. ###
+    @time @safetestset "Basic DSL" begin include("dsl/dsl_basics.jl") end
+    @time @safetestset "DSL Model Construction" begin include("dsl/dsl_model_construction.jl") end
+    @time @safetestset "Custom CRN Functions" begin include("dsl/custom_functions.jl") end
+    @time @safetestset "DSL Options" begin include("dsl/dsl_options.jl") end
+
+    ### Non-DSL model cration and modication. ###
+    @time @safetestset "ReactionSystem Components Based Creation" begin include("programmatic_model_creation/component_based_model_creation.jl") end
+    @time @safetestset "Programmatic Model Expansion" begin include("programmatic_model_creation/programmatic_model_expansion.jl") end
+
+    # Runs various miscellaneous tests.
+    @time @safetestset "API" begin include("miscellaneous_tests/api.jl") end
+    @time @safetestset "Symbolic Stoichiometry" begin include("miscellaneous_tests/symbolic_stoichiometry.jl") end
```

Comment:
Call these higher level functionality instead of miscellaneous?

Reply:
Aren't there some other categories (like visualization) that could also be considered higher level functionality? I just wanted an all-encompassing category for stuff I didn't know where to put.

Reply:
If you want I could change it to higher level though